.release-please-manifest.json

```diff
@@ -1,3 +1,4 @@
 {
-  ".": "0.2.36"
+  ".": "0.2.36",
+  "apis": "0.2.36"
```

Comment:
we should decide what should be the first version of the APIs package.

Reply:
I think same version as the current package makes sense...?

Reply:
This will slightly change how the release process works. @toddbaert and I have dealt with this enough times that it shouldn't be an issue here.